pkg/hooks/loader.go

```diff
@@ -174,14 +189,20 @@ func (h *Hook) SetTcsMocks(m []*models.Mock) {
 func (h *Hook) SetConfigMocks(m []*models.Mock) {
 	h.configMocks.deleteAll()
 	for index, mock := range m {
+		h.persistentMatchedMocks[mock.Name] = mock
 		mock.TestModeInfo.SortOrder = index
```

Comment:
why is this needed?

Reply:
Without is `persistentMatchedMocks` will contain only the tcsMocks